### PR TITLE
Fix global custom functionality

### PR DESCRIPTION
**Pull request by SimonWoolf**
This is a bit embarrassing...

I've just gotten around to moving back onto mainline rollbax from my fork (now that the global custom thing had been merged into master, https://github.com/elixir-addicts/rollbax/pull/46). And found it doesn't work.

Turns out it was completely my fault - one of the last-minute tweaks I made in that PR before it was merged actually broke it (by putting the `custom` field in the outermost map rather than in the `data` member), and the tests I added didn't catch it (as they just checked that the data was somewhere in there with a regex, rather than parsing it). And I hadn't noticed till now as I was still on my original version.

This fixes that and makes the test actually test it. (Edit: tests moved to https://github.com/elixir-addicts/rollbax/pull/55)

Sorry about that :disappointed:

### CODE REVIEW

File: test/rollbax/client_test.exs [hunk 1]
```diff
@@ -23,8 +23,7 @@ defmodule Rollbax.ClientTest do
     assert body =~ ~s("environment":"test")
     assert body =~ ~s("level":"warn")
     assert body =~ ~s("body":"pass")
-    assert body =~ ~s("foo":"bar")
-    assert body =~ ~s("qux":"custom")
+    assert Poison.decode!(body)["data"]["custom"] == %{"foo" => "bar", "qux" => "custom"}
```
Comment (whatyouhide) [inline test/rollbax/client_test.exs]: Let's keep matching with `=~` here instead of decoding. I know it didn't catch the bug in the first place and this is bad, but it's better to update all tests at once to use the parsed body IMO.

**Comment by whatyouhide** [comment]
No worries @SimonWoolf, happens to all of us :) Just left a tiny comment and we're good to go.

File: test/rollbax/client_test.exs (hunk 1 above)
Comment (SimonWoolf) [inline test/rollbax/client_test.exs]: > it's better to update all tests at once to use the parsed body IMO.

eh, I don't mind doing that, give me a few minutes

Comment (SimonWoolf) [inline test/rollbax/client_test.exs]: a684132

Comment (whatyouhide) [inline test/rollbax/client_test.exs]: @SimonWoolf I meant doing that in the whole test suite actually, so I would prefer to do that on a separate PR so that this PR remains very focused and tiny :) Would that be a problem for you?

Comment (SimonWoolf) [inline test/rollbax/client_test.exs]: sure, no prob

**Comment by whatyouhide** [comment]
Thanks @SimonWoolf! 💟
